frostsnap_coordinator/src/persist.rs

```diff
@@ -27,6 +27,21 @@ impl<T> Persisted<T> {
         Ok(ret)
     }
 
+    pub fn mutate2<C, R>(
```

Comment:
Maybe `mutate_with_update`?

Reply:
they all have an "update". This is intended to be the one we use. `staged_mutate` and `mutate` should be deleted.